bo/produktpaket_test.go

```diff
@@ -0,0 +1,80 @@
+package bo_test
+
+import (
+	"encoding/json"
+	"github.com/corbym/gocrest/is"
+	"github.com/corbym/gocrest/then"
+	"github.com/go-playground/validator/v10"
+	"github.com/hochfrequenz/go-bo4e/bo"
+	"github.com/hochfrequenz/go-bo4e/com"
+	"github.com/hochfrequenz/go-bo4e/enum/botyp"
+	"github.com/hochfrequenz/go-bo4e/enum/produktcode"
+	"reflect"
+	"testing"
+)
+
+var produktpaket = bo.Produktpaket{
+	Geschaeftsobjekt: bo.Geschaeftsobjekt{
+		BoTyp:           botyp.PRODUKTPAKET,
+		VersionStruktur: "1.1",
+	},
+	PaketId: 1,
+	Konfigurationen: []com.Produktkonfiguration{
+		{
+			Code:              produktcode.MESSTECHNISCHE_EINORDNUNG,
+			Eigenschaft:       produktcode.INTELLIGENTES_MESSSYSTEM,
+			Zusatzeigenschaft: "",
+		},
+		{
+			Code:              produktcode.BILANZKREIS,
+			Eigenschaft:       produktcode.BILANZKREIS,
+			Zusatzeigenschaft: "4000",
+		},
+	},
+	Prioritaet:           1,
+	MussVollstaendigSein: false,
+}
+
+// Test_Produktpaket_Deserialization deserializes an Produktpaket json
+func Test_Produktpaket_Deserialization(t *testing.T) {
```

Comment:
ist das das Casing in go für Funktionen?

Reply:
Zumindest bei uns für Tests 😅